projects/js-packages/components/components/alert/index.tsx

```diff
@@ -45,7 +56,7 @@ const getIconByLevel = level => {
  * @param {React.Component} props.children - The alert content.
  * @returns {React.Component}                The `Alert` component.
  */
-function Alert( { level, children, showIcon } ) {
+function Alert( { level, children, showIcon }: AlertProps ): React.ReactElement {
```

Comment:
Can you follow @manzoorwanijk suggestion and use `React.ReactNode`?

Reply:
yes https://github.com/Automattic/jetpack/pull/24204/commits/8f2d9f3ee69fead448221ff4eb4dfcfcfae2d4b5